.github/workflows/create-release-tag.yml

```diff
@@ -20,8 +20,8 @@ jobs:
           # BE AWARE --> Updating to a new MAJOR version will delete deprecated versions on a nightly schedule.
           # See https://github.com/Energinet-DataHub/.github#release-procedure for details
           major_version: 1
-          minor_version: 9
-          patch_version: 4
+          minor_version: 10
```

Comment:
Is this a major release? Behavior changes with this update.

Reply:
Agree. To be honest, I tried to avoid a major release because of the "BE AWARE" above, but I will ask Outlaws about this. :D

Reply:
The risk is on the side of datamigrations...